common/middleware/set-move-with-summary.js

```diff
@@ -0,0 +1,11 @@
+const moveHelpers = require('../helpers/move')
+
+function setMoveWithSummary(req, res, next) {
+  res.locals = {
+    ...moveHelpers.getMoveWithSummary(req.move),
+    ...res.locals,
+  }
+  next()
+}
+
+module.exports = setMoveWithSummary
```

Comment:
Sorry, I think I've missed where this common middleware is being used?

Reply:
Could this live in `/common/middleware/framework/` seeing as its related to the framework controllers?

Reply:
I put it there... but it's not technically got anything to do with the framework per se so moved it to the `/common/middleware` jumble. Maybe a move directory is waving to us over the horizon?